.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You missed this, I added it so I could review. Be careful with what you're committing.